app/views/feed/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "layouts/account", title: t("account.navigation.feed") do %>
+  <% if current_user.favorites.any? %>
+    <h2 class="feed__heading">New updates</h2>
+
+    <% if @feed_last_visited_at && @revisions.first.created_at < @feed_last_visited_at %>
```

Comment:
```suggestion
    <% if @feed_last_visited_at && (@revisions&.first.blank? || @revisions.first.created_at < @feed_last_visited_at) %>
```
to avoid a potential issue with there being no valid `@revisions.first`